.github/workflows/herdstat.yaml

```diff
@@ -32,5 +32,5 @@ jobs:
         with:
           author_name: cs-minion
           author_email: rng_cr_carbynestack@bosch.com
-          add: 'contribution-graph.svg'
+          add: 'profile/contribution-graph.svg'
```

Comment:
This not the new file created by the nightly `herdstat` run but the one that will exist in the `profile` folder as a result of merging your PR.

Reply:
Right. I also updated [.herdstat.yaml](https://github.com/carbynestack/.github/pull/12/files#diff-50eab34b05ede56ab86c6ec547baf3dcb7f22601e39b04122a8617dc7a060e09)